.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+align = true
+assumeStandardLibraryStripMargin = true
+maxColumn = 100
+
+# For better code clarity
+danglingParentheses = true
+
+docstrings = JavaDoc
+
+rewrite.rules = [
+  AsciiSortImports,
+  AvoidInfix,
+]
+
+# This also seems more idiomatic to include whitespace in import x.{ yyy }
+spaces.inImportCurlyBraces = true
+
+trailingCommas = always
```

Comment:
what does that do? 

Is this adding a trailing comma in line in...

```
Seq(
  "foo",
  "bar",
)
```

I'm fine with it, but this will be a problem for cross-compiling to 2.11.

Reply:
And if I remember correct, we will keep cross-compiling to 2.11, at least for 2.7.0. That said, `play-grpc` will need to compile to 2.11 as well.

Reply:
* as I understand it Scala 2.11 can be dropped now in 2.7.x
* IMO play-grpc doesn't need to start supporting 2.11
* but if need be we can change this to "preserve"